src/MongoDB.Driver/Linq/Linq3Implementation/LinqProviderV3.cs

```diff
@@ -61,7 +60,8 @@ internal sealed class LinqProviderV3 : LinqProvider
             IBsonSerializerRegistry serializerRegistry,
             ExpressionTranslationOptions translationOptions)
         {
-            throw new NotImplementedException();
+            // TODO: implement using LINQ3 instead of falling back to LINQ2
+            return LinqProvider.V2.TranslateExpressionToBucketOutputProjection(valueExpression, outputExpression, documentSerializer, serializerRegistry, translationOptions);
```

Comment:
For the time being falling back on LINQ2 for this.

What's blocking this is the pending implementation of the AstPipelineOptimizer to move accumulator expressions back to the $group stage.